clients/ops/admin-ui/src/features/privacy-requests/RequestTable.tsx

```diff
@@ -65,7 +66,14 @@ const RequestTable: React.FC<RequestTableProps> = () => {
     isFetching,
   } = useRequestTable();
 
-  return (
+  useEffect(() => {
```

Comment:
This resolved a React memory leak at runtime when a user clicks the User icon to logout. Memory leak warning was visible via browser console.